code/contact.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
```

Comment:
Hi! I'm Estefanía and i'll be taking a look at your site today so that we can leave it as functional and shiny as possible.👋😊